src/net/sourceforge/kolmafia/utilities/ChoiceUtilities.java

```diff
@@ -31,6 +32,23 @@ public class ChoiceUtilities {
     Pattern.compile("whichchoice=(\\d+)"),
   };
 
+  private static final Pattern[] PARSE_PATTERNS = {FORM_PATTERN, LINK_PATTERN};
+
+  private static final String CHOICE_PHP = "choice.php";
```

Comment:
I don't see the value of having this private constant. I'd prefer to actually see "choice.php" being tested in the code rather than having to look up the constant, obvious as it might be.